src/include/wt_internal.h

```diff
@@ -506,6 +506,7 @@ typedef uint64_t wt_timestamp_t;
 #include "dhandle.h"      /* required by btree.h, connection.h */
 #include "timestamp.h"    /* required by reconcile.h */
 #include "thread_group.h" /* required by rollback_to_stable.h */
+#include "verbose.h"      /* required by rollback_to_stable.h */
```

Comment:
Compilation didn't fail previously, but there is a dependency between `rollback_to_stable.h` and `verbose.h`. With this change VSCode can now jump from the uses of `WT_DECL_VERBOSE_MULTI_CATEGORY` in `rollback_to_stable.h` to the definition in `verbose.h`